validation/tdr_latest.xsd

```diff
@@ -61,7 +61,13 @@
   </xs:simpleType>
   <xs:complexType name="AttributeList-CT">
     <xs:sequence>
-      <xs:element name="attr" maxOccurs="unbounded"/>
```

Comment:
Let's revert this part and let the monolith enforce this during load for now.  I'm not sure we understand all the implications and I don't want to change it on both sides quiet yet.

Reply:
Sure. I'll remove this part.